webextensions/browser-compat-data.json

```diff
@@ -726,10 +726,10 @@
     }, 
     "connectNative": {
       "Chrome": {
-        "support": true
+        "support": "26"
```

Comment:
Also "28" according to the Chrome docs: https://developer.chrome.com/extensions/runtime#method-connectNative